TombEngine/Game/camera.cpp

```diff
@@ -323,6 +332,131 @@ void MoveCamera(GameVector* ideal, int speed)
 	}
 }
 
+
+void ObjCamera(ItemInfo* CamSlotId, int CamMeshID, ItemInfo* TargetItem, int TargetMeshID, bool cond)
+{
+	GameVector from, to;
+
+	//CamSlotId and TargetItem stay the same object until I know how to expand TargetItem to another object
```

Comment:
Have a space after `//` and finish the sentence with a `.` when adding comments. Keeps things tidy. :)